_data/strings.yml

```diff
@@ -19,3 +19,9 @@ locales:
     language_name: "中文"
     language: "官话"
     qmk_firmware: "QMK固件"
+
+  hi: &DEFAULT_HI
+    <<: *DEFAULT_EN
+    language_name: "हिंदी"
+    language: "भाषा"
+    qmk_firmware: "QMK-फर्मवेयर"
```

Comment:
Key-value pairs for `made_possible`, `hosted_on`, and `description` are also needed here.

See below for the English text.

https://github.com/qmk/qmk.fm/blob/27bc9ef1e3de79a21d387d4298201fb985dc3b4f/_data/strings.yml#L2-L8